src/js/edu-benefits/schema/edu-benefits.js

```diff
@@ -0,0 +1,396 @@
+        },
+        street2: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 50
+        },
+        city: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 51
+        }
+      },
+      required: [
+        'street',
+        'city',
+        'country'
+      ]
+    },
+    date: {
+      format: 'date',
+      type: 'string'
+    },
+    dateRange: {
+      type: 'object',
+      properties: {
+        from: {
+          $ref: '#/definitions/date'
+        },
+        to: {
+          $ref: '#/definitions/date'
+        }
+      },
+      required: ['from', 'to']
+    },
+    fullName: {
+      type: 'object',
+      properties: {
+        salutation: {
+          type: 'string'
+        },
+        first: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 30
+        },
+        middle: {
+          type: 'string'
+        },
+        last: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 30
+        },
+        suffix: {
+          'enum': options.suffixes
+        },
+      },
+      required: [
+        'first',
+        'last'
+      ]
+    },
+    phone: {
+      type: 'string',
+      pattern: '^[0-9]{10}$'
+    },
+    ssn: {
+      type: 'string',
+      pattern: '^[0-9]{9}$'
+    },
+  },
+  properties: {
+    chapter33: {
+      type: 'object',
+      properties: {
+        enabled: {
+          type: 'boolean'
+        },
+        electionType: {
+          type: 'string'
+        },
+        electionDate: {
+          $ref: '#/definitions/date'
+        }
+      }
+    },
+    chapter30: {
+      type: 'boolean'
+    },
+    chapter1606: {
+      type: 'boolean'
+    },
+    chapter32: {
+      type: 'boolean'
+    },
+    fullName: {
+      $ref: '#/definitions/fullName'
+    },
+    gender: {
+      type: 'string',
+      'enum': ['M', 'F']
+    },
+    birthday: {
+      $ref: '#/definitions/date'
+    },
+    socialSecurityNumber: {
+      $ref: '#/definitions/ssn'
+    },
+    address: {
+      $ref: '#/definitions/address'
+    },
+    phone: {
+      $ref: '#/definitions/phone'
+    },
+    secondaryPhone: {
+      $ref: '#/definitions/phone'
+    },
+    emergencyContact: {
+      type: 'object',
+      properties: {
+        fullName: {
+          $ref: '#/definitions/fullName'
+        },
+        sameAddressAndPhone: {
+          type: 'boolean'
+        },
+        address: {
+          $ref: '#/definitions/address'
+        },
+        phone: {
+          $ref: '#/definitions/phone'
+        },
+      }
+    },
+    bankAccount: {
+      type: 'object',
+      properties: {
+        accountType: {
+          type: 'string',
+          'enum': ['checking', 'savings']
+        },
+        bankName: {
+          type: 'string'
+        },
+        routingNumber: {
+          type: 'string'
+        },
+        accountNumber: {
+          type: 'string'
+        }
+      }
+    },
+    previouslyFiledClaimWithVa: {
```

Comment:
Looking at VONAPP, this question expands into a bunch of checkboxes asking what type of claim and the file number:
![screen shot 2016-08-22 at 4 11 03 pm](https://cloud.githubusercontent.com/assets/634932/17869854/1386c090-6883-11e6-996a-eabb54050669.png)